dali/CMakeLists.txt

```diff
@@ -132,20 +146,22 @@ endif()
 
 # Copy all headers from DALI_INST_HDRS list to DALI_WHEEL_DIR using install command
 # with `-D` option, that recursively creates missing directories in destination path
-add_custom_target(install_headers ALL
-    DEPENDS dali dali_operators
-)
-
-# Process the DALI_INST_HDRS list
-foreach(INSTALL_HEADER ${DALI_INST_HDRS})
-  file(RELATIVE_PATH HEADER_RELATIVE ${PROJECT_SOURCE_DIR} ${INSTALL_HEADER})
+if (BUILD_PYTHON)
```

Comment:
Don't we want to install headers for kernels somehow even if Python is not build.

Reply:
install headers yes, but not in the wheel dir. I have to check if `make install` works as expected

Reply:
make install works as expected